Sources/AST/ASTVisitor/ASTVisitor.swift

```diff
@@ -32,37 +32,51 @@ public struct ASTVisitor {
+      processResult.element = .contractDeclaration(
+        processResult.combining(visit(contractDeclaration, passContext: processResult.passContext)))
     case .structDeclaration(let structDeclaration):
-      processResult.element = .structDeclaration(processResult.combining(visit(structDeclaration, passContext: processResult.passContext)))
+      processResult.element = .structDeclaration(
+        processResult.combining(visit(structDeclaration, passContext: processResult.passContext)))
     case .enumDeclaration(let enumDeclaration):
-      processResult.element = .enumDeclaration(processResult.combining(visit(enumDeclaration, passContext: processResult.passContext)))
+      processResult.element = .enumDeclaration(
+        processResult.combining(visit(enumDeclaration, passContext: processResult.passContext)))
     case .traitDeclaration(let traitDeclaration):
-      processResult.element = .traitDeclaration(processResult.combining(visit(traitDeclaration, passContext: processResult.passContext)))
+      processResult.element = .traitDeclaration(
+        processResult.combining(visit(traitDeclaration, passContext: processResult.passContext)))
     }
 
-    let postProcessResult = pass.postProcess(topLevelDeclaration: processResult.element, passContext: processResult.passContext)
-    return ASTPassResult(element: postProcessResult.element, diagnostics: processResult.diagnostics + postProcessResult.diagnostics, passContext: postProcessResult.passContext)
+    let postProcessResult = pass.postProcess(topLevelDeclaration: processResult.element,
+                                             passContext: processResult.passContext)
+    return ASTPassResult(element: postProcessResult.element,
+                         diagnostics: processResult.diagnostics + postProcessResult.diagnostics,
+                         passContext: postProcessResult.passContext)
   }
 
-  func visit(_ contractDeclaration: ContractDeclaration, passContext: ASTPassContext) -> ASTPassResult<ContractDeclaration> {
+  func visit(_ contractDeclaration: ContractDeclaration,
+             passContext: ASTPassContext) -> ASTPassResult<ContractDeclaration> {
     var processResult = pass.process(contractDeclaration: contractDeclaration, passContext: passContext)
 
-    processResult.element.identifier = processResult.combining(visit(processResult.element.identifier, passContext: processResult.passContext))
+    processResult.element.identifier = processResult.combining(visit(processResult.element.identifier,
+                                                                     passContext: processResult.passContext))
 
-
-    processResult.passContext.contractStateDeclarationContext = ContractStateDeclarationContext(contractIdentifier: contractDeclaration.identifier)
+    processResult.passContext.contractStateDeclarationContext =
+      ContractStateDeclarationContext(contractIdentifier: contractDeclaration.identifier)
```

Comment:
Would a small constant declaration be preferable to this line break?

Reply:
Such as

```
let x = ContractStateDeclarationContext(contractIdentifier: contractDeclaration.identifier)
processResult.passContext.contractStateDeclarationContext = x
```

?

We don't really think there's a need to add an additional constant to the scope just to assign a variable. Splitting this across two lines with sufficient indentation (as we have done) is quite readable.

Reply:
I did mean the other way round. So x = ....

More because we frequently use this latter with respect to passContext properties.